update.sh

```diff
@@ -3,13 +3,15 @@ set -e
 
 declare -A gpgKeys
 gpgKeys=(
+	[7.0]='1A4E8B7277C42E53DBA9C7B9BCAA30EA9C0D5763'
```

Comment:
True, but we aren't able to get the fullVersion from the packagesUrl anyway (which means a manual update for every bump, and limiting that to just changing the `7.0/Dockerfile` rather than the script feels safer) and we currently don't need to put the gpg key in.